### PR TITLE
Have custom visualization on by default

### PR DESCRIPTION
**Pull request by jingzhang36**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2891)

### CODE REVIEW

File: manifests/kustomize/base/pipeline/ml-pipeline-ui-deployment.yaml [hunk 1]
```diff
@@ -24,4 +24,6 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: ALLOW_CUSTOM_VISUALIZATIONS
+            value: "true"
```
Comment (rmgogogo) [inline manifests/kustomize/base/pipeline/ml-pipeline-ui-deployment.yaml]: please also cover the gcp_marketplace

Comment (jingzhang36) [reply manifests/kustomize/base/pipeline/ml-pipeline-ui-deployment.yaml]: done.

**Comment by rmgogogo** [comment]
how about add it as predefined type to avoid customization?
https://github.com/kubeflow/pipelines/tree/master/backend/src/apiserver/visualization/types

worried about the possible bad codes which may trigger big issue (e.x. deleted data).

@IronPan

**Pull request merged**